app_confetti/fetch/file.py

```diff
@@ -0,0 +1,12 @@
+import dotenv
+
+
+def fetch_to_env(override=True):
```

Comment:
Consider injecting the start path instead of `usecwd` and filename here, makes it easier to switch envs if you wish to.

Reply:
You can't dotenv takes an exact path to load, or loads from cwd (if usecwd is true), otherwise it tries to derive the location from `__file__` which will lead to trying to go from the installed location of app_confetti which is less desirable.

Reply:
Aaah, sorry I got mixed up, you can define what file it should look for, it defaults to ".env", you should be able to pass a variable in to allow people to define which file they're looking for, if you were to have a `test.dev`, `prod.dev` etc